client/src/components/settings/exam-token.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import { Button, Panel, Modal, Spacer } from '@freecodecamp/ui';
+import { useTranslation } from 'react-i18next';
+import { FullWidthRow } from '../helpers';
+import { generateExamToken } from '../../utils/ajax';
+
+function ExamToken(): JSX.Element {
+  const [examToken, setExamToken] = useState<string | null>(null);
+  const [examTokenError, setExamTokenError] = useState<string | null>(null);
+
+  const [recentlyGenerated, setRecentlyGenerated] = useState(false);
+  const [generatedExamToken, setGeneratedExamToken] = useState(false);
+
+  const [copySuccess, setCopySuccess] = useState<string | null>(null);
+  const [copyError, setCopyError] = useState<string | null>(null);
+
+  const debouncedSetRecentlyGenerated = () => {
+    setRecentlyGenerated(true);
+    setTimeout(() => setRecentlyGenerated(false), 10000);
+  };
```

Comment:
```suggestion
```
We might as well inline this, since it's just two lines quite descriptive code. Also, it's not a debounce, it's just a temporary delay.